src/main/java/io/github/jhipster/domain/util/JSR310DateConverters.java

```diff
@@ -82,6 +82,28 @@ public ZonedDateTime convert(Date source) {
         }
     }
 
+    public static class StringToZonedDateTimeConverter implements Converter<String, ZonedDateTime> {
+        public static final StringToZonedDateTimeConverter INSTANCE = new StringToZonedDateTimeConverter();
+
+        private StringToZonedDateTimeConverter() {
+        }
+
+        public ZonedDateTime convert(String source) {
+            return source == null ? null : ZonedDateTime.parse(source);
+        }
+    }
+
+    public static class ZonedDateTimeToStringConverter implements Converter<ZonedDateTime, String> {
+        public static final ZonedDateTimeToStringConverter INSTANCE = new ZonedDateTimeToStringConverter();
+
+        private ZonedDateTimeToStringConverter() {
+        }
+
+        public String convert(ZonedDateTime source) {
```

Comment:
Shouldn't it be `public static String convert(ZonedDateTime source)`?

Reply:
No, it should not.